### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by ademyavuz42**
https://masterbranchacademy.atlassian.net/browse/SCRUM-66

Note: configuration done

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,100 @@
...
+
+> * e2e ## : Test files and data.
+> * e2e/tests/: Test files.
+> * support/commands.js: Custom commands.
+> * support/index.js: Support file.
+
+
+### Writing Tests <hr>
+1. Create a new test file in cypress/integration/tests/:
+> // cypress/integration/tests/exampleTest.spec.js <br>
```
Comment (MasterBranchAcademy) [inline README.md]: Cypress is not supporting integration folder anymore, instead its better to name is as "`e2e`"

File: README.md [hunk 2]
```diff
@@ -1,2 +1,100 @@
...
+import HomePage from '../../support/pageObjects/HomePage'; <br><br>
```
Comment (MasterBranchAcademy) [inline README.md]: Do you think its needed?

Comment (adiltashkentli) [inline README.md]: it's acceptable for beginning

File: cypress/fixtures/example.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (MasterBranchAcademy) [inline cypress/fixtures/example.json]: do we use this data anywhere?

File: cypress/fixtures/example.json [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
+  "name": "Using fixtures to represent data",
+  "email": "hello@cypress.io",
+  "body": "Fixtures are a great way to mock data for responses to routes"
+}
```
Comment (adiltashkentli) [inline cypress/fixtures/example.json]: we can delete these ones

File: package.json [hunk 5]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "mba-2024-cypress-automation",
+  "version": "1.0.0",
+  "description": "cypress practice 2024",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (MasterBranchAcademy) [inline package.json:7]: "test": "npx cypress open"

Comment (adiltashkentli) [inline package.json:7]: you can use best practice

Comment (MasterBranchAcademy) [reply package.json:7]: "adem": "npx cypress open"

**Review by MasterBranchAcademy — APPROVED** [review]
LGTM!

File: cypress.config.js [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+      on('before:browser:lunch', (browser = {}, launchOptions) => {
+        if (browser.name === 'chrome') {
+          launchOptions.args.push('--disable-dev-shm-usage');
+          return launchOptions;
+        }
+      })
+    },
+
+  },
+});
```
Comment (adiltashkentli) [inline cypress.config.js:28]: Lgtm

File: package.json [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+  },
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "cypress": "^13.0.0"
+  }
+}
```
Comment (adiltashkentli) [inline package.json:14]: Looks good

**Pull request merged**